### PR TITLE
GitHub workflow: Don't run Request review for external PRs

### PR DESCRIPTION
**Pull request by pavel-mikula-sonarsource**
This fixes issue that appeared when the action was triggered for PR from external fork.

### CODE REVIEW

File: .github/workflows/RequestReview.yml [hunk 1]
```diff
@@ -73,23 +75,26 @@ jobs:
...
-                    console.log("Moving card");
-                    github.projects.moveCard({ card_id: card.id, position: "bottom", column_id: REVIEW_IN_PROGRESS_COLUMN });
-                } else if (issue.pull_request) {
-                    console.log("Creating PR card");
-                    github.projects.createCard({ column_id: REVIEW_IN_PROGRESS_COLUMN, content_id: context.payload.pull_request.id, content_type: "PullRequest" });
-                } else {
-                    console.log("Creating Issue card");
-                    github.projects.createCard({ column_id: REVIEW_IN_PROGRESS_COLUMN, content_id: issue.id, content_type: "Issue" });
+                if(issue.state == "open") {
+                  removeAssignees(issue);
```
Comment (andrei-epure-sonarsource) [inline .github/workflows/RequestReview.yml:79]: if there is a 2nd reviewer added, I believe we shouldn't reset+add, but add another face on the kanban, to reflect the people that should have a look

**Review by andrei-epure-sonarsource — CHANGES_REQUESTED** [review]

File: .github/workflows/RequestReview.yml (hunk 1 above)
Comment (pavel-mikula-sonarsource) [reply .github/workflows/RequestReview.yml:79]: Let's stick with baby steps here as well. Review state is very hard to read from API. I can't get list of reviewers with "yellow dot in UI".

Comment (pavel-mikula-sonarsource) [reply .github/workflows/RequestReview.yml:79]: I added a note about this idea to my Kanban backlog.

Comment (andrei-epure-sonarsource) [reply .github/workflows/RequestReview.yml:79]: sure, it's an edge case, if it's not easy to fix, we leave it for later

**Review by andrei-epure-sonarsource — APPROVED** [review]
LGTM

**Pull request merged**